src/libstd/unstable/dynamic_lib.rs

```diff
@@ -45,12 +45,22 @@ impl Drop for DynamicLibrary {
 }
 
 impl DynamicLibrary {
+    // FIXME (#12938): Until DST lands, we cannot decompose &str into
```

Comment:
Yeah this can just be removed now, we've since dealt with the underlying issue in one way or nother!